src/store/modules/topology/index.ts

```diff
@@ -643,10 +643,15 @@ const actions: ActionTree<State, any> = {
         for (const call of topoCalls) {
           if (call.detectPoints.includes('CLIENT')) {
             clientIdsC.push(call.id);
-          } else {
+          }
+          if (call.detectPoints.includes('SERVER')) {
             serverIdsC.push(call.id);
           }
         }
+        if (!serverIdsC.length || !clientIdsC.length) {
```

Comment:
```suggestion
        if (!serverIdsC.length && !clientIdsC.length) {
```
Should use `&&`? And will you build request by the matrix of `serverIdsC` and `clientIdsC`?

Reply:
>  And will you build request by the matrix of `serverIdsC` and `clientIdsC`?

Yes

Reply:
> Should use `&&`

Here I need to change the logic.